homeassistant/components/fivem/binary_sensor.py

```diff
@@ -0,0 +1,41 @@
+"""The FiveM binary sensor platform."""
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import FiveMDataUpdateCoordinator, FiveMEntity
+from .const import DOMAIN, ICON_STATUS, NAME_STATUS
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the FiveM binary sensor platform."""
+    server = hass.data[DOMAIN][entry.entry_id]
```

Comment:
```suggestion
    coordinator = hass.data[DOMAIN][entry.entry_id]
```

Am I correct that you pass the coordinator? Since it isn't typed (and probably using a wrong var name) it is hard to understand.

Reply:
Yes, this should be the `coordinator`